recipe/recipe.yaml

```diff
@@ -91,6 +106,9 @@ tests:
       - ./13-boosted_top < data/boosted_top_event.dat
       - $CXX 14-groomers.cc -o 14-groomers $CXXFLAGS $LDFLAGS -lfastjettools -lfastjet
       - ./14-groomers < data/boosted_top_event.dat
+      - ${{ PYTHON }} -c "import fastjet_cxx; print(fastjet_cxx)"
```

Comment:
We can formalize this as well in the test infrastructure with the following

```yaml
tests:
  - python:
      imports:
        - fastjet_cxx
      pip_check: true
      python_version: ${{ python }}
```

where

```yaml
python_version: ${{ python }}
```

should work by picking up the value defined in the `.ci_support/` config files. e.g. for `.ci_support/osx_arm64_python3.12.____cpython.yaml`

```yaml
python:
- 3.12.* *_cpython
```

Reply:
ah that's much cleaner - thank you - sorry for stepping on your toes there.

Reply:
No no you're good!